### PR TITLE
feat: add corvid-xpert for debugging jsons

### PR DESCRIPTION
**Pull request by tomenden**
Usage:
`corvid-xpert [command]`

This decodes the JSONs so the fs contains unobfuscated data.

### CODE REVIEW

File: packages/corvid-local-site/src/utils/wixFiles.js [hunk 1]
```diff
@@ -1,12 +1,38 @@
 const { prettyStringify } = require("./prettify");
+const isString_ = require("lodash/isString");
+const flow_ = require("lodash/flow");
+const atob = require("atob");
```
Comment (tombenezra) [inline packages/corvid-local-site/src/utils/wixFiles.js]: personally I wouldn't use an npm module but rather create a util function

File: packages/corvid-local-site/src/utils/wixFiles.js [hunk 2]
```diff
@@ -1,12 +1,38 @@
...
+
+const decodeContent = payload => {
+  const decode = flow_([atob, escape, decodeURIComponent, JSON.parse]);
+  return isString_(payload)
+    ? decode(payload)
+    : { ...payload, content: decode(payload.content) };
+};
+const encodeContent = content => {
+  const encode = flow_([JSON.stringify, encodeURIComponent, unescape, btoa]);
+  return !content.content
```
Comment (tombenezra) [inline packages/corvid-local-site/src/utils/wixFiles.js]: it's here to differentiate between json and string files? ain't we wrap everything with content?

File: packages/corvid-local-site/src/utils/wixFiles.js (hunk 1 above)
Comment (tomenden) [reply packages/corvid-local-site/src/utils/wixFiles.js]: Care to explain why?
I believe the library does exactly what you would write on your own

File: packages/corvid-local-site/src/utils/wixFiles.js (hunk 2 above)
Comment (tomenden) [reply packages/corvid-local-site/src/utils/wixFiles.js]: Unfortunately, currently the obfuscated content is sometimes located in `content` but in other times (page files) it is located in `content.content`.
I think we should change that

**Pull request merged**